lxd/container_lxc.go

```diff
@@ -1264,6 +1273,44 @@ func (c *containerLXC) startCommon() (string, error) {
 					return "", err
 				}
 			}
+		} else if m["type"] == "nic" {
+			if m["nictype"] == "bridged" && shared.IsTrue(m["security.mac_filtering"]) {
```

Comment:
Since this is only valid on veths, should we reject it if someone tries to set it on a macvlan or something?